pom.xml

```diff
@@ -270,6 +261,31 @@
 
       <build>
         <plugins>
+          <plugin>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
```

Comment:
Is it always enabled (i.e., when I run tests locally)?

Reply:
No, expected behaviour is that it is only enabled when `ci-build` profile is active.
I really want to test it all, but I guess we can't unless we stop skipping tests (coverage report isn't prepared if we skip all tests). So this will have to wait until fixing that `LD_LIBRARY_PATH` issue.

Reply:
That's what I hoped for - 0% coverage just to be sure that we did everything and integrated with coveralls correctly. But when we skip test, we don't get report with 0% - we don't get any report at all and coveralls plugin doesn't do anything :(
I'll try to use this https://github.com/exonum/exonum-java-binding/pull/336 for now and try to connect with coveralls locally.